Restructurize docs (#31)
asyncapi/asyncapi-react: README.md

    The `theme` property is optional and contains styles for specific parts of the AsyncApi component. For information on how to change styles, read the [Theme Modification](./docs/configuration/theme-modification.md) document.

    > **NOTE:** If you do not pass the [**disableDefaultTheme**](./docs/configuration/config-modification.md) config as `true`, this property is concatenated with the [default theme](./library/src/theme/default.ts)

  - **config?: Partial<ConfigInterface\>**

    The `config` property is optional and contains configuration for the AsyncApi component. For more information on the available configuration options, read the [Configuration Modification](./docs/configuration/config-modification.md) document.
    This property is concatenated with the [default configuration](./library/src/config/default.ts).

    > **NOTE:** The `Partial<T>` type means that every field in the `T` type is optional.

## Development

For information on how to set up a development environment, write and run tests, follow the naming and architecture convention defined for the project in the [Development Guide](./docs/development/guide.md).

## Releasing

> For information on how to release new version of library and/or playground app, and create changelog, please read [Releasing](./docs/development/releasing.md) document.

Reviewer: ```suggestion
For information on how to release a new version of the library or the playground application, or details on creating changelog, read the [Releasing](./docs/development/releasing.md) document.
```
